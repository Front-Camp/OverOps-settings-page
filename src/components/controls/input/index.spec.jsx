import * as React from 'react';
import {shallow} from 'enzyme/build';
import Input from './index';

describe('Input', () => {
  test('Component should be rendered correctly', () => {
    const checkbox = shallow(<Input />);

    // Snapshot demo
    expect(checkbox).toMatchSnapshot();
  });
});
