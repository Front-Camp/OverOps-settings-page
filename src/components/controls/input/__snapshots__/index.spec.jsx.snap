// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component should be rendered correctly 1`] = `
<input
  className="input"
/>
`;
